### PR TITLE
Create and Update order

### PR DESCRIPTION
**Pull request by britnay268**
## Description
- Created apis to create and update an order
- Create a file for form to create and update order
- Edit event listeners to allow the form to 
     - Show when 'create an order' is pressed on the Nav bar or on the webpage 
     - Creates an order when the form is submitted.
     - Edit an existing order when the edit link is clicked on an order

## Related Issue
#33 

## Motivation and Context
This changed is necessary to allow a user to create an order via the button in the navigation bar or the button on the webpage as well as edit an order if a mistake was made when creating the order.

## How Can This Be Tested?
When a user is logged in, they can click 'Create an Order' from the navigation bar and type in information required in the form and submit the form.

THEN, the user will see existing orders if any and the new order that was created.

These steps are the same when clicking 'Create an Order' button the webpage.

If the user wants to edit an order, the user can click the edit link on the card and the form with pop up for the user the edit existing details then when 'Update Order' is clicked, the user will see the updated order with the changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: components/forms/newOrderForm.js
Comment (jessefrench) [inline components/forms/newOrderForm.js:1]: You need to swap the placeholder text for your email and phone inputs

Comment (britnay268) [reply components/forms/newOrderForm.js:1]: Committed the changes!

**Review by yarelismartin — APPROVED** [review]
👍

**Review by AndrewSpur73 — APPROVED** [review]
💯

**Pull request merged**